### PR TITLE
Create web_cve_2022_27925_exploit.yml

### PR DESCRIPTION
**Pull request by danielgottt**

### CODE REVIEW

**Comment by frack113** [comment]
from the tests results : 
Rule rules/web/web_cve_2022_27925_exploit.yml has the selection (selection_noauth) with a list of only 1 element in detection
Rule rules/web/web_cve_2022_27925_exploit.yml has a title field with too many characters (>70)
Rule rules/web/web_cve_2022_27925_exploit.yml has a title that has not title capitalization. Words: 'collaboration, suite, email, server, unauthenticated, remote, command, injection'

**Comment by danielgottt** [comment]
Thank you for clarifying those issues frack. Was also able to add some additional items.

File: rules/web/web_cve_2022_27925_exploit.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+tags:
+  - attack.initial_access
+  - attack.t1190
+  - cve.2022.27925
+logsource:
+    category: webserver
+detection:
+    selection_servlet:
+        cs-method: 'POST'
+        c-uri|contains: '/service/extension/backup/mboximport?'
```
Comment (frack113) [inline rules/web/web_cve_2022_27925_exploit.yml:20]: https://github.com/SigmaHQ/sigma/wiki/Specification#escaping
`c-uri|contains: '/service/extension/backup/mboximport\?'`

File: rules/web/web_cve_2022_27925_exploit.yml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        cs-cookie: 'ZM_AUTH_TOKEN'
+    selection_status:
+        sc-status:
+            - '401'
+            - '200'
+    selection_shell:
+        cs-uri|contains: '/zimbraAdmin/'
+        cs-uri|endswith: '.jsp'
+        cs-status|contains: '200'
+    condition: all of selection* or selection_shell
```
Comment (frack113) [inline rules/web/web_cve_2022_27925_exploit.yml:36]: `all of selection*` is a AND , we can put in the same selection
```yaml
selection_servlet:
        cs-method: 'POST'
        c-uri|contains: '/service/extension/backup/mboximport\?'
        c-uri|contains|all:
            - 'account-name'
            - 'ow'
            - 'no-switch'
            - 'append'
        cs-cookie: 'ZM_AUTH_TOKEN'
        sc-status:
            - '401'
            - '200'
```

File: rules/web/web_cve_2022_27925_exploit.yml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    selection_servlet:
+        cs-method: 'POST'
+        c-uri|contains: '/service/extension/backup/mboximport?'
+    selection_uri:
+        c-uri|contains|all:
+            - 'account-name'
+            - 'ow'
+            - 'no-switch'
+            - 'append'
+        cs-cookie: 'ZM_AUTH_TOKEN'
```
Comment (frack113) [inline rules/web/web_cve_2022_27925_exploit.yml:27]: `cs-cookie` not sure we have this in the log.
If the ref you get  a "normal" web  log: 
[attacker_ip] - - [30/Jun/2022:05:33:18 +0000] "POST [snipped]/service/extension/backup/mboximport?account-name=admin&ow=1&no-switch=true HTTP/1.0" 401 299 "[snipped]" "Mozilla/5.0 (X11; Linux x86_64; rv:101.0) Gecko/20100101 Firefox/101.0"

Comment (danielgottt) [reply rules/web/web_cve_2022_27925_exploit.yml]: This is a good point. I ended up stripping the field.

**Pull request merged**